base/logger/NEWS.md

```diff
@@ -1,3 +1,7 @@
+# PEcAn.logger (development version)
```

Comment:
```suggestion
# PEcAn.logger 1.8.1.9000
```

Push back if you like yours better, but I've been doing other NEWS files this way -- I have a vague memory there was some package-build-time magic that required the top heading to to match the package version. Don't remember what the magic was or what would break if the version was wrong, though...